engine/orchestration/src/main/java/com/cloud/vm/VirtualMachineManagerImpl.java

```diff
@@ -1047,6 +1049,13 @@ public void orchestrateStart(final String vmUuid, final Map<VirtualMachineProfil
             resourceCountIncrement(owner.getAccountId(),new Long(offering.getCpu()), new Long(offering.getRamSize()));
         }
 
+        // Increment the VR resource count if the global setting is set to true
+        // and resource.count.running.vms is also true
```

Comment:
As the comment was intended to be a literal description of the following line, it is not necessary.